src/doc/data_into_visit/PlainTextFormat.rst

```diff
@@ -48,38 +94,69 @@ Here are the first 10 lines of an example of a file representing 3D points.
+Here is the Python code to plot this data in VisIt_
+
+.. literalinclude:: ../../test/tests/databases/plaintext.py
+   :language: Python 
+   :start-after: # Points plot {
+   :end-before: # Points plot }
+
+and the resulting data plotted in VisIt_
+
+.. figure:: ../../../test/baseline/databases/plaintext/PlainText_Points.png
+
+Defining a single variable on a 2D uniform grid
 -----------------------------------------------
 
-The variable is interpreted as a node centered variable and the X and Y coordinates are the indexes into the rows and columns.
-The rows represent values along the X direction and the rows get stacked in the Y direction.
+The data is interpreted as a node centered variable on a uniform mesh where the row and column indices define the X and Y coordinates.
+The rows represent values along the X direction and the rows get *stacked* in the Y direction.
+Each row further *down* in the file gets stacked *up*, one upon the other in the visualized result in VisIt_.
+This means that the row-by-row *downward* direction in the file is the same as the *upward* (positive Y) direction in the visualized result in VisIt_.
 
 The first line can be an optional list of variable names.
-The first variable name will be used for the name of the variable.
-Note that you will need to provide the same number of variable names as there are columns, even though only the first one is used.
+The first column name will be used for the name of the variable.
+Other column names are ignored but nonetheless required to read the file properly. 
 The remaining lines consist of rows, where each row represents the values for a single Y coordinate.
 
 Here is an example of a file representing 3D points.
+In this example, the values on each row are separated by spaces.
 
-.. literalinclude:: data_examples/regulargrid2d.csv
+.. literalinclude:: data_examples/array.csv
 
 Here is the Python script that created the file.
 
-.. literalinclude:: data_examples/plain_text.py
-   :lines: 34-51
+.. literalinclude:: ../../test/tests/databases/plaintext.py
+   :language: Python 
+   :start-after: # Array gen {
+   :end-before: # Array gen }
 
-Here are the Plain Text reader options used to read the data.
+Here are the ``PlainText`` reader options used to read the data.
 
 .. figure:: images/PlainTextOptions3.png
 
 The columns for the X, Y and Z coordinates are not used.
+
+Here is the Python code to plot this data in VisIt_
+
+.. literalinclude:: ../../test/tests/databases/plaintext.py
+   :language: Python 
+   :start-after: # Array plot {
+   :end-before: # Array plot }
+
+and the resulting data plotted in VisIt_
+
+.. figure:: ../../../test/baseline/databases/plaintext/PlainText_2DArray.png
+
+Note that the reddest part of the plot (e.g. highest numerical values in the data) appears in the upper right corner of the plot whereas the highest numerical values in the file data row-by-row listing appears in the *lower* right corner.
```

Comment:
@biagas and @brugger1 this issue really can lead to confusion (for me at least). I wonder if it makes sense to have an `invert Y axis` option in the read options which is on by default for the `2D Array` case so that what you see in a file listing or excel spreadsheet is 1:1 with the plot? In fact, if this data is plotted in excel, do we get same orientation relative to VisIt or inverted? I guess I should check that.

Reply:
I think we should match what Excel does. If Excel plots it in the same manner, we can add the `invert Y axis` option. If Excel plots  with an implied `invert Y axis`, then we should change our reader and not bother with an `invert Y axis` option. My 2 cents worth.

Reply:
Ok, I plotted this data in excel. I wasn't able to match color map with VisIt but you get the idea. Below, the file listing is also color coded by a heat map. Of course, if the heat map chosen matches VisIt's color map, the resulting heat map coloring would be inverted from VisIt's plot. That said, when you create a 2D color contour of that data in Excel, Excel does invert the Y axis relative to the rows in the tabular data just like VisIt does. So, I say we don't worry about it other than the remarks here in our docs about it.
![Screen Shot 2022-03-14 at 10 38 51 AM](https://user-images.githubusercontent.com/5720676/158229872-4d618d49-f73a-4cde-99f2-d090d67caf5d.png)

Reply:
I agree.